transport-native-epoll/src/main/java/io/netty/channel/unix/DomainSocketChannel.java

```diff
@@ -30,4 +31,6 @@
 
     @Override
     DomainSocketChannelConfig config();
+
+    PeerCredentials peerCredentials() throws IOException;
```

Comment:
Agreed it is a breaking change. Although I don't think this will have a big impact on users as I don't think many (or any) people will implement this interface. I'll defer to your preference (my preference is to just leave it on the interface).